doc/ci.md

```diff
@@ -1,16 +1,18 @@
 # CI/CD Pipeline
 
-## Restrictions
+## Prerequisites (Schema)
```

Comment:
Please don't start with this, the user should first get information about how to use the CI, especially `jenkins build please` should be mentioned.